src/test/java/com/openlattice/chronicle/ChronicleServerTests.java

```diff
@@ -1,11 +1,497 @@
+import com.google.common.collect.SetMultimap;
+import com.google.common.eventbus.EventBus;
+import com.openlattice.chronicle.configuration.ChronicleConfiguration;
+import com.openlattice.chronicle.data.ChronicleAppsUsageDetails;
+import com.openlattice.chronicle.services.ChronicleService;
+import com.openlattice.chronicle.services.ChronicleServiceImpl;
+import com.openlattice.client.ApiClient;
+import com.openlattice.client.RetrofitFactory;
+import com.openlattice.data.DataApi;
+import com.openlattice.data.DeleteType;
+import com.openlattice.data.requests.NeighborEntityDetails;
+import com.openlattice.edm.EdmApi;
+import com.openlattice.entitysets.EntitySetsApi;
+import com.openlattice.search.SearchApi;
+import com.openlattice.search.requests.EntityNeighborsFilter;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+
+import static com.openlattice.edm.EdmConstants.ID_FQN;
 
 public class ChronicleServerTests {
 
+    protected static final Logger logger                  = LoggerFactory.getLogger( ChronicleServerTests.class );
+
+    // property type FQNS
+    private static final FullQualifiedName FULL_NAME_FQN   = new FullQualifiedName( "general.fullname" );
+    private static final FullQualifiedName DURATION        = new FullQualifiedName( "general.Duration" );
+    private static final FullQualifiedName DATE_LOGGED_FQN = new FullQualifiedName( "ol.datelogged" );
+    private static final FullQualifiedName START_DATE_TIME = new FullQualifiedName( "ol.datetimestart" );
+    private static final FullQualifiedName TITLE_FQN       = new FullQualifiedName( "ol.title" );
+    private static final FullQualifiedName DATETIME_FQN    = new FullQualifiedName( "ol.datetime" );
+    private static final FullQualifiedName USER_FQN        = new FullQualifiedName( "ol.user" );
+    private static final FullQualifiedName RECORD_TYPE_FQN  = new FullQualifiedName( "ol.recordtype" );
+
+    private static final String                       STUDY_ES_NAME             = "chronicle_study";
+    private static final String                       DEVICES_ES_NAME           = "chronicle_device";
+    private static final String                       APP_DATA_ES_NAME          = "chronicle_app_data";
+    private static final String                       RECORDED_BY_ES_NAME       = "chronicle_recorded_by";
+    private static final String                       CHRONICLE_USER_APPS       = "chronicle_user_apps";
+    private static final String                       USED_BY_ES_NAME           = "chronicle_used_by";
```

Comment:
These are already defined in `ChronicleServiceImpl`. Please refactor so these are only defined once. There's a `constants` directory in the project.

Reply:
Dropping tests for now.